### PR TITLE
adding util.sh with docker+help utils

### PR DESCRIPTION
**Pull request by miki725**

### CODE REVIEW

File: util.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    echo -e ${ORANGE}Flags:${END_COLOR}
+    echo
+    {
+        echo -h/--help print this message | _help_format
+        _help_flags $@
+    } | sort
+    echo
+
+    echo -e ${ORANGE}Commands:$END_COLOR
```
Comment (orangematt) [inline util.sh]: Can you make `$var` consistent either with or without the braces? My personal preference is for with the braces, especially in the cases here where inserting the colors is harder to read without the braces, but I'm fine with whatever as long as it's consistent. Not just in this function, but throughout.

File: util.sh [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            | head -n1
+    )
+    echo ${service:-$default}
+}
+
+# ============================================================================
+# HELP
+# ============================================================================
+
+ORANGE="\033[0;33m"
```
Comment (orangematt) [inline util.sh]: nit: this is yellow, not orange

File: util.sh (hunk 1 above)
Comment (miki725) [reply util.sh]: adjusted. just left `$@` and couple of other standalong places without braces but all strings I think adjusted to use `${}`

File: util.sh (hunk 2 above)
Comment (miki725) [reply util.sh]: haha. copy-paste across repos for a while was using this as orange

Comment (miki725) [reply util.sh]: adjusted

**Review by orangematt — DISMISSED** [review]

**Review by orangematt — APPROVED** [review]

**Pull request merged**